SignaturePad/wwwroot/sigpad.min.js

```diff
@@ -261,15 +269,15 @@ export default class Sigpad {
         if (this._element.width != this._element.clientWidth) {
             this._element.width = this._element.clientWidth;
         }
-        var ctx = this._element.getContext("2d");
-        ctx.clearRect(0, 0, this._element.width, this._element.height);
-        this._applyOptions(ctx);
-        this._mousePosis.filter((stroke) => stroke.length > 2).forEach((stroke) => drawPoints(ctx, stroke));
+        if (this._drawing) {
+            var ctx = this._element.getContext("2d");
+            ctx.clearRect(0, 0, this._element.width, this._element.height);
```

Comment:
clearing the canvas deletes a previous existing signature. Can you clarify why you clear the canvas here?

Reply:
because i'd draw multiple times over the canvas. if u can confirm that it doesnt create lag we can remove the clearing. or redraw the signature. seems to be an artifact though

Reply:
i dont know how the canvas handles drawn lines if it disregards objects and draws pixels it shouldnt lag becaus it doesnt stack